<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui_treebank_parser</class>
 <widget class="QMainWindow" name="gui_treebank_parser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui_treebank_parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QTabWidget" name="inputTabSelector">
      <property name="toolTip">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="treebankTab">
       <attribute name="title">
        <string>Single treebank</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Treebank File</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputTreebankFile">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FileChooserButton" name="inputTreebankButton">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Heads File</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputTreebankFile">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FileChooserButton" name="outputTreebankButton">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>65</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="treebankCollectionTab">
       <attribute name="title">
        <string>Treebank collection</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Treebank collection</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputTreebankCollectionFile">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FileChooserButton" name="inputTreebankCollectionButton">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Output directory</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputTreebankCollectionDirectory">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FileChooserButton" name="outputTreebankCollectionButton">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="sentenceConsistencyCheckBox">
          <property name="toolTip">
           <string extracomment="Ensure a sentence is written to the output only if none of the equivalent sentences in the other treebanks have been discarded."/>
          </property>
          <property name="toolTipDuration">
           <number>5000</number>
          </property>
          <property name="text">
           <string>Keep consistency of sentences</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>36</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Choose a format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TreebankFormatComboBox" name="treebankFormatSelector">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Actions available for format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="availableActionList">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="AddRemoveActionButton" name="actionAddButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="AddRemoveActionButton" name="actionRemoveButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Actions to be applied</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ActionsTable" name="chosenActionTable">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Option</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value Type</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="RunParserButton" name="runTreebankParser">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lalReleaseCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Use laloptimized</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lalDebugCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Use lal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="toolTip">
           <string>Output logging messages showing the progress of the script. The higher the debugging level the more messages will be displayed.
* 0: display only 'error' and 'critical' messages.
* 1: messages from 0 plus 'warning' messages.
* 2: messages from 1 plus 'info' messages.
* 3: messages from 2 plus 'debug' messages.</string>
          </property>
          <property name="text">
           <string>        Logging messages level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="loggingLevelSpinBox">
          <property name="maximum">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="msgLogger">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Clear log messages</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="HelpMenu" name="only_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="how_to"/>
    <addaction name="about"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="only_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="how_to">
   <property name="text">
    <string>How to</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionq1">
   <property name="text">
    <string>q1</string>
   </property>
  </action>
  <action name="actionq2">
   <property name="text">
    <string>q2</string>
   </property>
  </action>
  <action name="check_LAL_reachable">
   <property name="text">
    <string>Is LAL reachable?</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreebankFormatComboBox</class>
   <extends>QComboBox</extends>
   <header>gui.actions.TreebankFormatComboBox</header>
  </customwidget>
  <customwidget>
   <class>FileChooserButton</class>
   <extends>QPushButton</extends>
   <header>gui.FileChooserButton</header>
  </customwidget>
  <customwidget>
   <class>AddRemoveActionButton</class>
   <extends>QPushButton</extends>
   <header>gui.actions.AddRemoveActionButton</header>
  </customwidget>
  <customwidget>
   <class>ActionsTable</class>
   <extends>QTableWidget</extends>
   <header>gui.actions.ActionsTable</header>
  </customwidget>
  <customwidget>
   <class>RunParserButton</class>
   <extends>QPushButton</extends>
   <header>gui.RunParserButton</header>
  </customwidget>
  <customwidget>
   <class>HelpMenu</class>
   <extends>QMenu</extends>
   <header>gui.HelpMenu</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>msgLogger</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1039</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
